src/AVFoundation/AVLayerVideoGravity.cs

```diff
@@ -104,7 +108,8 @@ partial class AVPlayer
 	{
 		public AVLayerVideoGravity? ExternalPlaybackVideoGravity {
 			set {
-				WeakExternalPlaybackVideoGravity = value.HasValue ? AVPlayerLayer.EnumToKey (value.Value) : null;
+				if (value.HasValue && AVPlayerLayer.EnumToKey (value.Value) is NSString val)
+					WeakExternalPlaybackVideoGravity = val;
```

Comment:
```suggestion
				WeakExternalPlaybackVideoGravity = value.HasValue ? AVPlayerLayer.EnumToKey (value.Value) : null!;
```

Reply:
How about `WeakExternalPlaybackVideoGravity = value.HasValue ? AVPlayerLayer.EnumToKey (value.Value)! : null!;`

Reply:
Yeah, that works too.

Reply:
@mandel-macaque This is the comment I am referring to below